Source/Common/std string.cpp

```diff
@@ -241,3 +241,31 @@ std::wstring stdstr::ToUTF16 ( bool * bSuccess)
 	}
 	return res;
 }
+
+std::wstring stdstr::ToUTF16(uint32_t CodePage, bool * bSuccess)

```

Comment:
I like the change to use the code page to get the correct internal name, needs to also be applied to the rom information dialog.

But I dislike duplicating ToUTF16 can we not just add the CodePage param to the current version with a defualt of CP_UTF8?